docs/client-guide.adoc

```diff
@@ -155,6 +161,8 @@ Currently, only the core functionality of making software updates available and
 
 To communicate with the SOTA Client over unix domain sockets, `socket = true` must be defined in the `[gateway]` section of the config file. Additionally, the names of the two sockets are configurable in the link:#__network[[network]] section.
 
+The SOTA Client will create the commands socket, but the software loading manager must create the events socket. Note that it must be readable and writable by root, and be at the location configured in `sota.toml`.
```

Comment:
While it is true that some external software will need to create the events socket to listen to events, I'm not sure I'd use the Software Loading Manager name here as this implies the GENIVI version which is designed to be used via D-Bus.